src/main.rs

```diff
@@ -556,7 +558,8 @@ fn transclude_dry_run(
 }
 
 fn transclude(parser: &MdParser, file_name: &Path) -> Fallible<Document> {
-    let source_main = fs::read_to_string(&file_name)?;
+    let source_main = fs::read_to_string(&file_name)
```

Comment:
Since this is done multiple times, maybe a function in `util` like
```rust
pub fn read_file(path: &Path) -> Fallible<String> {
  read_to_string(&path).map_err(|err| format!("{}: {}", err, path.display()))
}
```
would be useful?